src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -21,6 +21,62 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
```

Comment:
Sure, that makes sense for this particular generation of hardware, but it would be good not to have to remember this and fix it later if it changes. And, if it does, then it would have to be implemented this way (querying the number of threads) anyways if it were to be backwards compatible.